### PR TITLE
Added slight clarification to "Error to be aware of"

### PR DESCRIPTION
**Pull request by stevedsimkins**
Saw a lot of users on the discord struggling with this, tried to make it flash in people's faces a bit more and clarify the problem they're facing!

### CODE REVIEW

File: Solana_NFTs/en/Section_2/Lesson_3_Depoy_NFTs_To_Devnet.md [hunk 1]
```diff
@@ -124,7 +124,7 @@ Arweave is pay once, store **forever**. They do this using an [algorithm](https:
 
 You may be asking yourself — "well, who's paying to host my stuff then!?". Well if you look through the source code of the script [here](https://github.com/metaplex-foundation/metaplex/blob/59ab126e41e6d85b53c79ad7358964dadd12b5f4/js/packages/cli/src/helpers/upload/arweave.ts#L93), you'll see that Metaplex pays for it themselves for now to help you out!
 
-### 😡 **Error to be aware of.**
+## 😡 **Error to be aware of.**
```
Comment (adilanchian) [inline Solana_NFTs/en/Section_2/Lesson_3_Depoy_NFTs_To_Devnet.md:127]: Can we keep this as h3?

**Review by adilanchian — CHANGES_REQUESTED** [review]
Appreciate the call out! 

Just one change please :)

File: Solana_NFTs/en/Section_2/Lesson_3_Depoy_NFTs_To_Devnet.md (hunk 1 above)
Comment (stevedsimkins) [reply Solana_NFTs/en/Section_2/Lesson_3_Depoy_NFTs_To_Devnet.md]: done!

**Comment by stevedsimkins** [comment]
Made the changes; ready to merge!

**Comment by adilanchian** [comment]
Thanks for this @stevedsimkins !

**Pull request merged**